src/Symfony/Component/Notifier/Bridge/Smsapi/README.md

```diff
@@ -7,12 +7,13 @@ DSN example
 -----------
 
 ```
-SMSAPI_DSN=smsapi://TOKEN@default?from=FROM
+SMSAPI_DSN=smsapi://TOKEN@default?from=FROM&fast=FAST
```

Comment:
I propose to rename the parameter "priority"

Reply:
For clarity for sms api users I wanted to keep the same name as sms api uses in the docs. https://www.smsapi.com/docs/#2-single-sms.

@OskarStark What do you think? Should I change it to "priority" or keep "fast"?

Reply:
fast=1 then?